include/intercept.h

```diff
@@ -55,7 +57,7 @@ intercept_wrapper();
 void
 setup_intercepts();
 
-size_t
-my_call_comp(size_t, char *, void *);
+// size_t
```

Comment:
Commented code.

Reply:
Same as the comment for `intercept.c`. I can also add a `TODO` here, if it's worth it.

Reply:
A `TODO` would be good, thanks.